sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -187,6 +188,42 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Creates a notification when an Application Offering Change Request is completed by the ministry.
+   * @param notification input parameters to generate the notification.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param entityManager optional entity manager to execute in transaction.
+   */
+  async saveApplicationOfferingChangeRequestCompletedByMinistry(
```

Comment:
This name is not related to "create a notification". It sounds like it is saving the offering change request instead.

Reply:
Maybe not for this PR but the method name in line 161 also have this problem (missing "Notification" in the end of the name)

Reply:
@andrepestana-aot  I believe that must be part of the PR.

Reply:
Fixed in both the places.